packages/flutter/lib/src/material/radio.dart

```diff
@@ -264,18 +264,20 @@ class Radio<T> extends StatefulWidget {
   /// [ThemeData.focusColor] is used.
   final Color? focusColor;
```

Comment:
Just noticed this is missing, is that expected?

Reply:
Yes, mentioned this in the description section:) The radio button cannot show focused state when the RadioListTile requests focus, because the ListTile will take focus. The `focusColor` is also skipped in the `SwitchListTile` and `RadioListTile` PRs.